### PR TITLE
game context menu: add misc things

### PR DESCRIPTION
**Pull request by Zangetsu38**
- Add Open folder for game and savedata
- Add Update History on game context menu.
- Add game information.
- Refactor order/name for have same real psvita hw (set real psvita element after emulate element).

# Result:

- Context menu:
image

- Update history:
image

- Information:
image

### CODE REVIEW

File: vita3k/gui/src/game_context_menu.cpp [hunk 1]
```diff
@@ -52,6 +55,63 @@ void delete_game(GuiState &gui, HostState &host, const std::string &title_id) {
...
+
+    for (auto &i : update_history_info) {
+        const auto startpos = "<";
+        const auto endpos = ">";
+
+       if (i.find_first_of('\n') != std::string::npos)
+           i.erase(i.begin() + i.find_first_of('\n'));
+
+       while (i.find("</li> <li>") != std::string::npos)
+           if (i.find("</li> <li>") != std::string::npos)
```
Comment (1whatleytay) [inline vita3k/gui/src/game_context_menu.cpp]: Doesn't while statement guarentee this?

File: vita3k/gui/src/game_context_menu.cpp [hunk 2]
```diff
@@ -52,6 +55,63 @@ void delete_game(GuiState &gui, HostState &host, const std::string &title_id) {
...
+              i.replace(i.find("</li> <li>"), i.find(endpos) + 2 - i.find(startpos), "\n");
+       while (i.find("<li>") != std::string::npos)
+           if (i.find("<li>") != std::string::npos)
```
Comment (1whatleytay) [inline vita3k/gui/src/game_context_menu.cpp]: Ditto.

File: vita3k/gui/src/game_context_menu.cpp [hunk 3]
```diff
@@ -52,6 +55,63 @@ void delete_game(GuiState &gui, HostState &host, const std::string &title_id) {
...
+                i.replace(i.find("<li>"), i.find(endpos) + 1 - i.find(startpos), ". ");
+       while (i.find(startpos) != std::string::npos)
+           if (i.find(">") + 1 != std::string::npos)
+                i.erase(i.find(startpos), i.find(endpos) + 1 - i.find(startpos));
+
+        bool seen_space = false;
```
Comment (1whatleytay) [inline vita3k/gui/src/game_context_menu.cpp]:
```suggestion
        bool found_space = false;
```

File: vita3k/gui/src/game_context_menu.cpp (hunk 1 above)
Comment (Zangetsu38) [reply vita3k/gui/src/game_context_menu.cpp]: Is spécial, need while and if

File: vita3k/gui/src/game_context_menu.cpp (hunk 2 above)
Comment (Zangetsu38) [reply vita3k/gui/src/game_context_menu.cpp]: is same for all, i have learn how that works with learn hpo strip tag, and without while or if is broken

**Comment by Zangetsu38** [comment]
Rebase and refactor pr with add open game folder

**Pull request merged**